README.md

```diff
@@ -7,20 +7,28 @@ This is the POC of Jenkinsfile Runner Action for GitHub Actions in GSoC 2022.
   - [Introduction](#introduction)
   - [Pre-requisites](#pre-requisites)
   - [Inputs](#inputs)
-    - [Shared Inputs](#shared-inputs)
-    - [jfr-container-action Unique Inputs](#jfr-container-action-unique-inputs)
-    - [jfr-static-image-action Unique Inputs](#jfr-static-image-action-unique-inputs)
+    - [Container Actions Inputs](#container-actions-inputs)
```

Comment:
Are you generating the ToC automatically?

Reply:
Yes, I am.